src/screens/Settings/ChangePassword.js

```diff
@@ -0,0 +1,127 @@
+// Setup
+import React, { Component } from "react";
+import { StyleSheet, View } from "react-native";
+
+// Vendor
+import {
+  Container, Content, H1, H2, H3, Button, Text, Item, Label, Input,
+} from 'native-base';
+
+import { Formik } from "formik";
+
+// Redadalertas
+import { colors } from "styles";
+import userServices from "services/user";
+import { checkIfLoggedIn } from "utils/user";
+
+const styles = StyleSheet.create({
+  view: {
+    flex: 1,
+    flexDirection: "row",
+    justifyContent: "center",
+    alignItems: "stretch",
+    backgroundColor: "#F5FCFF",
+    color: "#000000"
```

Comment:
We should extract these two colors to our "styles" folder.

Reply:
Ok, doing that for the next PR!